github-actions/breaking-changes-label/lib/BUILD.bazel

```diff
@@ -16,7 +16,7 @@ ts_library(
     deps = [
         "//github-actions:utils",
         "//ng-dev/commit-message",
-        "//ng-dev/pr/common",
+        "//ng-dev/pr:config",
```

Comment:
Have you considered moving this into a `config/` folder instead? That is more aligned with what we do with `/release` and is more consistent overall with target naming.

Reply:
If you would like to we can, but I personally prefer not doing it that way.  We can do it in  a followup later if we end up wanting to.

Reply:
I think my main argument for this is that maintaining a target at the same level results in unnecessary extra code for excluding the `config.ts` file from the other glob. and I like the idea of a TS Bazel target matching just the directory where I can import it from (makes it easy to deal with deps). 

Either way works for me. Not feeling super strong but I prefer the other way haha 😄 We should clean up either release or pr IMO, but a follow-up works!

Reply:
I went ahead and did it the way you outlined as I realized that my primary argument for doing it this way was around not relying on importing from a directories `index` file, but we sorted out that problem in the repo and rely on this throughout the repo now.

Reply:
👍 cool, thanks for doing that change!